tests/test_directmessagepricechanged.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python
+#
+# A library that provides a Python interface to the Telegram Bot API
+# Copyright (C) 2015-2025
+# Leandro Toledo de Souza <devs@python-telegram-bot.org>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser Public License for more details.
+#
+# You should have received a copy of the GNU Lesser Public License
+# along with this program.  If not, see [http://www.gnu.org/licenses/].
+"""This module contains an object for testing a Direct Message Price."""
+
+from typing import TYPE_CHECKING
+
+import pytest
+
+from telegram import DirectMessagePriceChanged, User
+from tests.auxil.slots import mro_slots
+
+if TYPE_CHECKING:
+    from telegram._utils.types import JSONDict
+
+
+@pytest.fixture
+def direct_message_price_changed():
+    return DirectMessagePriceChanged(
+        are_direct_messages_enabled=True, direct_message_star_count=100
```

Comment:
```suggestion
        are_direct_messages_enabled=DirectMessagePriceChangedTestBase.are_direct_messages_enabled,
        direct_message_star_count=DirectMessagePriceChangedTestBase.direct_message_star_count,
```

to ensure that asserts in `test_to_dict()` come from the same source